support-frontend/assets/components/paymentButton/defaultPaymentButtonContainer.tsx

```diff
@@ -77,10 +80,6 @@ export function DefaultPaymentButtonContainer({
 	const currency = currencies[currencyId];
 	const amountWithCurrency = simpleFormatAmount(currency, amount);
 
-	const { countryId } = useContributionsSelector(
-		(state) => state.common.internationalisation,
-	);
```

Comment:
No idea how this worked in the first place as it was required before it was declared.